internal/pkg/template/templates/addons/s3/env/access_policy.yml

```
@@ -0,0 +1,47 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: Your workload's name.
+Resources:
+  {{logicalIDSafe .Name}}AccessPolicy:
```

Similar feedback as above for this file
```suggestion
  {{logicalIDSafe .Name}}BucketAccessPolicy:
```